### PR TITLE
Minify Player/DiscretePlayer with scale_buttons/visible_buttons/visible_loop_options

### PR DESCRIPTION
**Pull request by ahuang11**
Make it a little less bulky

```python
import panel as pn

pn.extension()
pn.widgets.Player(name='test', button_scale=0.8, width=300).servable()
```

image

button_scale=1
image

```python
discrete_player = pn.widgets.DiscretePlayer(name='Player', visible_buttons=["play", "pause"], scale_buttons=0.9, show_loop_controls=False, width=150)
discrete_player
```

image

### CODE REVIEW

**Comment by philippjfr** [comment]
Needs a merge/rebase it looks like.

**Comment by ahuang11** [comment]
Ok rebased it!

File: panel/tests/ui/widgets/test_player.py [hunk 1]
```diff
@@ -53,3 +53,45 @@ def test_name_and_show_value(page):
...
+    assert not page.is_visible(".reverse")
+    assert not page.is_visible(".first")
+    assert not page.is_visible(".previous")
+    assert not page.is_visible(".next")
+    assert not page.is_visible(".last")
+    assert not page.is_visible(".slower")
+    assert not page.is_visible(".faster")
+
+    player.visible_buttons = ["first"]
+    wait_until(lambda: page.is_visible(".first"))
```
Comment (philippjfr) [inline panel/tests/ui/widgets/test_player.py:71]: `wait_until` is meant for waiting for some condition in Python to be met. For checking things on the frontend you should use the `expect` API:

```suggestion
    expect(page.locator(".first")).to_be_visible()
```

File: panel/tests/ui/widgets/test_player.py [hunk 2]
```diff
@@ -53,3 +53,45 @@ def test_name_and_show_value(page):
...
+def test_player_visible_loop_options(page):
+    player = Player(visible_loop_options=["loop", "once"])
+    serve_component(page, player)
+
+    assert page.is_visible(".loop")
+    assert page.is_visible(".once")
+    assert not page.is_visible(".reflect")
+
+    player.visible_loop_options = ["reflect"]
+    wait_until(lambda: page.is_visible(".reflect"))
```
Comment (philippjfr) [inline panel/tests/ui/widgets/test_player.py:85]: Same here.

**Review by philippjfr — APPROVED** [review]
Two small comments on the tests, otherwise looks good!

File: panel/tests/ui/widgets/test_player.py (hunk 2 above)
Comment (ahuang11) [inline panel/tests/ui/widgets/test_player.py:85]:
```suggestion
    expect(page.locator(".reflect")).to_be_visible()
```

**Pull request merged**